boards/intel/aerofc-v1/init/rc.board_sensors

```diff
@@ -14,3 +14,6 @@ ist8310 -C -b 1 -R 4 start
 aerofc_adc start
 
 ll40ls start i2c
+
+# Internal SPI
+ms5611 -s start
```

Comment:
I need to double check this one.

Reply:
@dagar sure, I will wait for answer to re-push

Reply:
ping @dagar

Reply:
ping @dagar

Reply:
Sorry, was on a technology free vacation.

Reply:
@dagar I didn't get clear answer about that one, so kept it for now